pikaur/prompt.py

```diff
@@ -41,7 +41,7 @@ def read_answer_from_tty(question: str, answers: str = 'Yn') -> str:
         if ord(answer) == 13:
             answer = default
             return default
-        if answer in answers:
+        if answer in [a.lower() for a in answers]:
```

Comment:
`answers.lower()`?

Reply:
🤦forgot answers isn't a list despite looking straight at it.